src/seedsigner/views/seed_views.py

```diff
@@ -1499,6 +1499,8 @@ def run(self):
         ).display()
 
         if selected_menu_num == RET_CODE__BACK_BUTTON:
+            # Need to pop extra view from backstack to get to ScanView
+            self.controller.back_stack.pop()
```

Comment:
A `View` shouldn't assert control over the `Controller`'s `BackStack`.

Probably instead we need to add `skip_current_view=True` to all the `Destination` instances in `AddressVerificationStartView`.

Any `View` that does that kind of auto-routing needs to skip itself as a possible "back" `Destination`.

Reply:
Removed this update and added skip_current_view=True as suggested.